### PR TITLE
Oculus Integration

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
## Overview

Oculus Integration initial load

### CODE REVIEW

File: XRTK.Oculus/Packages/com.xrtk.oculus/Controllers/OculusRemoteController.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        }
+
+        /// <inheritdoc />
+        public override MixedRealityInteractionMapping[] DefaultInteractions => new[]
+        {
+            new MixedRealityInteractionMapping(0, "Button.DpadUp", AxisType.Digital, DeviceInputType.ButtonPress, "DpadUp"),
+            new MixedRealityInteractionMapping(1, "Button.DpadDown", AxisType.Digital, DeviceInputType.ButtonPress, "DpadDown"),
+            new MixedRealityInteractionMapping(2, "Button.DpadLeft", AxisType.Digital, DeviceInputType.ButtonPress, "DpadLeft"),
+            new MixedRealityInteractionMapping(3, "Button.DpadRight", AxisType.Digital, DeviceInputType.ButtonPress, "DpadRight"),
+            new MixedRealityInteractionMapping(4, "Button.One", AxisType.Digital, DeviceInputType.ButtonPress, "One"),
```
Comment (provencher) [inline XRTK.Oculus/Packages/com.xrtk.oculus/Controllers/OculusRemoteController.cs]: Button.One seems like weird naming

**Comment by provencher** [comment]
This seems nice and clean! Nice job

**Comment by provencher** [comment]
@SimonDarksideJ could you clarify a little a bit on how the OculusApi is obtained and defined? I can't seem to see any clear calls to the oculus SDK itself. How are you bridging this data?

File: XRTK.Oculus/Packages/com.xrtk.oculus/Controllers/OculusRemoteController.cs (hunk 1 above)
Comment (SimonDarksideJ) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Controllers/OculusRemoteController.cs]: Talk to Oculus, it's all their fault.
Like with OpenVR, we have kept the interaction naming to match the underlying API's definitions for what something is.

**Comment by SimonDarksideJ** [comment]
It's all in the hand waving
[ XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs](https://github.com/XRTK/Oculus/pull/2/files#diff-7511f54db8ea0c6a9f0ae9b6b5fad662)

**Comment by SimonDarksideJ** [comment]
Nudge @StephenHodgson , will the pipeline work now?

**Pull request merged**